compoments/radioFieldSet.js

```diff
@@ -73,6 +74,7 @@ class RadioFieldSet extends Component {
                 {this.title}
               </h1>
             </legend>
+            { this.hintText && <p id={`hint-text-${this.name}`}>{this.hintText}</p>}
```

Comment:
This isn't displayed in a lighter colour than the title as per the [prototype](https://housing-repairs-prototype.herokuapp.com/current/area-type).

The prototype uses/displays using the [GDS Design System](https://design-system.service.gov.uk) and we should use the appropriate CSS class to apply the correct styling. In this case the CSS class would be `govuk-hint`, see [this page](https://design-system.service.gov.uk/components/text-input/) (no anchor link, search for "Hint text").

The GDS Design System also provides guidance on tags and more to use, which I believe we should following.